pyroma/ratings.py

```diff
@@ -392,6 +392,16 @@ def message(self):
         return "You should have three or more owners of the project on PyPI."
 
 
+class TestSuitDeprecation(BaseTest):
+    weight = 50
+
+    def test(self, data):
+        return "test_suite" not in data
+
+    def message(self):
+        return "The `setup.py test` command and accompanying `test_suit` field has been deprecated."
```

Comment:
I'm not sure why this typo is everywhere, but the term and corresponding option is `test_suite`, not `test_suit` :laughing:

Reply:
*facepalm*

Reply:
*phew* I only made that mistake there. It gets to stay, as the test in pointless in the 4x branch...